src/cpp/rtps/builtin/discovery/endpoint/EDP.cpp

```diff
@@ -1027,7 +1027,7 @@ bool EDP::pairing_writer_proxy_with_local_reader(const GUID_t& local_reader, con
 bool EDP::pairing_remote_writer_with_local_reader_after_security(const GUID_t& local_reader,
                 const WriterProxyData& remote_writer_data)
 {
-    std::lock_guard<std::recursive_mutex> pguard(*mp_PDP->getMutex());
+    // std::lock_guard<std::recursive_mutex> pguard(*mp_PDP->getMutex());
```

Comment:
Do we want to keep this commented out code? (same above)

Reply:
both removed.